go/logic/applier.go

```diff
@@ -781,7 +782,7 @@ func (this *Applier) CreateAtomicCutOverSentryTable() error {
 }
 
 // AtomicCutOverMagicLock
-func (this *Applier) AtomicCutOverMagicLock(sessionIdChan chan int64, tableLocked chan<- error, okToUnlockTable <-chan bool, tableUnlocked chan<- error) error {
+func (this *Applier) AtomicCutOverMagicLock(sessionIdChan chan int64, tableLocked chan<- error, okToUnlockTable <-chan bool, tableUnlocked chan<- error, dropCutOverSentryTableOnce sync.Once) error {
```

Comment:
We must pass a pointer to `sync.Once` or else it is a copy of `sync.Once` which makes it a sync.Twice 😆

Reply:
okay, let me fix it.

Reply:
@shlomi-noach updated